marshal.go

```diff
@@ -2208,7 +2210,7 @@ func unmarshalTuple(info TypeInfo, data []byte, value interface{}) error {
 		return nil
 	}
 
-	return unmarshalErrorf("cannot unmarshal %s into %T", info, value)
+	return unmarshalErrorf("cannot unmarshal %s into %T. Accepted types: struct, []interface{}, array, slice.", info, value)
```

Comment:
Unmarshaler is also accepted I think. Also the accepted types other than `[]interface{}` have to be a pointer if I'm reading the code correctly (so `*struct, *array, *slice`)

Reply:
Good catch! Thank you.